src/Repository/LinkBlockRepository.php

```diff
@@ -384,4 +409,49 @@ private function getHookMaxPosition($idHook)
 
         return $qb->execute()->fetchColumn(0);
     }
+
+    /**
+     * @param int $linkBlockId
+     * @param array $shopIds
+     *
+     * @throws DatabaseException
+     */
+    private function updateShopAssociation(int $linkBlockId, array $shopIds): void
```

Comment:
I wish I could use `AbstractObjectModelHandler` here 🙄

Reply:
Maybe it means we were wrong when we built `AbstractObjectModelHandler` ? And we should have built something else that we could use here, using composition over inheritance?

Reply:
It would be possible to use it if I rewrite module to use commands right? but I'm not sure if it's worth it at this moment, for now I want to help  @matthieu-rolland to deliver what's important here and move to next tasks 

but if you guys think we should rewrite this module now - let me know

Reply:
Indeed it's not worth it for now.

Also I think Commands and Queries are meant to define the Core interface, so for Modules it sound a bit weird 🤔  to use them.

Reply:
so as discussed off github, we can use `AbstractObjectModelHandler` 👍